test/index.js

```diff
@@ -69,7 +69,7 @@ test('exact chain-of-words', (t) => {
     pull.filter((msg) => msg.value.content.text.toLowerCase().includes(INPUT)),
     pull.collect((err, msgs) => {
       t.error(err, 'no error');
-      t.equals(msgs.length, 4, 'four posts');
+      t.equals(msgs.length, 2, 'two posts');
```

Comment:
```bash
% rg -a 'labore officia exercitation' /tmp/ssb-search2/flume/log.offset|wc -l
2
```
(there are multiple messages on each line but i verified by looking at the results in `less`)

Reply:
tried finding a search with a few more results but i could only find searches with either 1-2 results or like 30+ (which would be harder to verify)